mgmtsystem_claim/views/res_partner_views.xml

```diff
@@ -17,7 +13,7 @@
                     context="{'search_default_partner_id': active_id, 'default_partner_id': active_id}"
                     name="%(mgmtsystem_claim_action)d"
                     icon="fa-comments"
-                    groups="sales_team.group_sale_salesman"
+                    groups="sales_team.group_sale_salesman,base.group_partner_manager"
```

Comment:
Don't add the `base.group_partner_manager` here. It'll change the previous behaviour.
I don't know why it was added to the view `groups_id` before, but only users belonging to `sales_team.group_sale_salesman` were able to see this button (as it was the only group defined in the element's `groups` attribute). So it shuold remain like this, IMO

Reply:
Following  the migration instruction, I deleted (I don't know why isn't reported)
```
<field
            name="groups_id"
            eval="[(4, ref('sales_team.group_sale_salesman')), (4, ref('base.group_partner_manager'))]"
        />
```
adding the "base.group_partner_manager" to the already present groups parameter.
I'm not confident with this change, so if you are sure, I trust you and delete it.

Reply:
Yes, please.
The explaination is this:


**Before:**

- The `ir.ui.view` was applied for both groups
- The button was shown only for `sales_team.group_sale_salesman`

**With your change:**

- The `ir.ui.view` is applied for all groups (normal, as Odoo deprecated the use of `groups_id`)
- The button is shown for both `sales_team.group_sale_salesman` and `base.group_partner_manager`


---

As you can see, the behaviour has changed.
To keep the previous behaviour we simply need to remove the `group_id` from the `ir.ui.view`, and keep the previous value for `groups` on the button element

Reply:
Thanks for explanation. ASAP I will fix tests using "remote" pr on crm.